### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by Metfriet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +42,21 @@ You will only need to run this command when dependencies change in `package.json
 
 We use npm scripts and [Webpack][] as our build system.
 
+### Microservices Overview
```
Comment (rnugraha) [inline README.md]: Could you maybe use this instead

> Podium is built in a microservices architecture which works in following way:
> 
> * Podium Gateway, serves the Angular application and also handles web traffic.
> * Podium UAA, is a User Account and Authentication server based on Spring Security. This server provides OAuth2 tokens for securing the gateway.
> * Podium-Registry, a runtime application on which all applications registers and get their configuration from. It also provides runtime monitoring dashboards.
>

File: README.md [hunk 2]
```diff
@@ -41,9 +42,21 @@ You will only need to run this command when dependencies change in `package.json
...
+2. Podium-UAA
+3. Podium-Gateway
 
 ### Running Podium for development
 
+First everything should be installed by running the following in the root folder.
+
+    mvn clean install
+
 Before the uaa and gateway services can start [Podium Registry][] needs to be up and running.
```
Comment (rnugraha) [inline README.md]: Pls provide link to podium registry repo

Comment (Metfriet) [reply README.md]: The links are provided at the bottom and filled in automatically

File: README.md (hunk 1 above)
Comment (Metfriet) [reply README.md]: I've added it but I feel we should still tell people the order in which to run this.

File: README.md [hunk 3]
```diff
@@ -41,9 +42,29 @@ You will only need to run this command when dependencies change in `package.json
 
 We use npm scripts and [Webpack][] as our build system.
 
+### Microservices Overview
+
+Podium is built in a microservices architecture which works in following way:
+
+**Podium Gateway**, serves the Angular application and also handles web traffic.
```
Comment (rnugraha) [inline README.md]: could you make it as list?

File: README.md [hunk 4]
```diff
@@ -7,13 +7,14 @@
 Podium is the request portal for samples, data and images from biobanks, with the purpose to uniformize the request 
 and review processes for all associated users and organisations.
 
-The application scaffold was generated using JHipster 4.0.0, you can find documentation and help at [https://jhipster.github.io/documentation-archive/v4.0.0](https://jhipster.github.io/documentation-archive/v4.0.0).
-
 This application is configured for Service Discovery and Configuration with the Podium-Registry. On launch, it will refuse to start if it is not able to connect to the Podium-Registry at [http://localhost:8761](http://localhost:8761).
```
Comment (rnugraha) [inline README.md]: suggestion to have Required Node.js and JDK versions

File: README.md [hunk 5]
```diff
@@ -62,30 +83,6 @@ Add the `help` flag on any command to see how you can use it. For example, `yarn
 
 The `yarn run` command will list all of the scripts available to run for this project.
```
Comment (rnugraha) [inline README.md]: Pls make it clear where to run yarn or mvn when we describing about uaa or gateway

File: README.md (hunk 1 above)
Comment (rnugraha) [reply README.md]: Then remove this part:

>  There are two services in this repository, the uaa service and the gateway service. There is a third the Podium Registry that has its own repository and needs to be running for the others to work.

Comment (Metfriet) [reply README.md]: it is gone now

**Pull request merged**